dev-tools/audius-cmd

```diff
@@ -24,7 +30,11 @@ for arg in "$@"; do
 	fi
 done
 
-turbo run build --filter=@audius/commands
+if [[ ! -f "$PROJECT_DIR/packages/commands/dist/index.js" ]]; then
+	turbo run build --filter=@audius/commands
+	# Give container time to sync output
+	sleep 1
```

Comment:
We build outside the container and the output is quite sizable. I was noticing that if you try to immediately run the docker command on the next line, it barfs about `unexpected end of input` because docker hasn't finished synchronizing the file into the `commands` container yet. 
Another reason why I'd like to get this out of docker soon :-)

Reply:
1s is enough sleep?

Reply:
In practice yes. If it isn't, you can just re-run audius-cmd and it won't do the build the second time and should succeed.